source/chart-table-builder-partials/chart-and-table-builders.md

```diff
@@ -0,0 +1,40 @@
+## Chartbuilder and Tablebuilder
+
+Chartbuilder and Tablebuilder function in roughly the same way. Chartbuilder will be detailed here and anything 
+particular to Tablebuilder should be minor.
+
+### Charts in the Publisher
+
+Charts in the Ethnicity Facts and Figures ecosystem work on three files:
+
+- `templates/cms/create_chart.html` is the template that contains the chartbuilder form.
+- `rd-chart-objects.js` is a factory for building rd-chart objects.
+- `rd-graph.js` is a renderer for rendering rd-chart objects.
```

Comment:
Full-stops on bullet end?

Reply:
OK I didn't apply the style guide, this was copy-and-paste job.

Will fix up.